src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -18,6 +18,8 @@
 @Component
 public class DataLoader
 {
+	
+	private static List<Map> placeCategories;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "placeCategories" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)